opentech/apply/funds/tests/test_views.py

```diff
@@ -608,3 +632,36 @@ def test_cant_access_normal_page(self):
     def test_cant_access_non_existing_page(self):
         response = self.get_page({'id': 555})
         self.assertEqual(response.status_code, 403)
+
+
+class TestSuperUserSubmissionView(BaseSubmissionViewTestCase):
+    user_factory = SuperUserFactory
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.submission = ApplicationSubmissionFactory()
+        super().setUpTestData()
+
+    def __setUp__(self):
+        self.refresh(self.submission)
+
+    def test_can_screen_submission(self):
+        screening_outcome = ScreeningStatusFactory()
+        self.post_page(self.submission, {'form-submitted-screening_form': '', 'screening_status': screening_outcome.id})
+        submission = self.refresh(self.submission)
+        self.assertEqual(submission.screening_status, screening_outcome)
+
+    def test_cant_screen_submission(self):
```

Comment:
Name is incorrect - `test_can_screen_applications_in_final_status`?

Reply:
... copy paste error strikes again, thanks for the catch!